phoenix-core/src/it/java/org/apache/phoenix/end2end/ReverseScanIT.java

```diff
@@ -52,11 +52,11 @@
 import com.google.common.collect.Maps;
 
 
-public class ReverseScanIT extends BaseHBaseManagedTimeIT {
-    protected static final String ATABLE_INDEX_NAME = "ATABLE_IDX";
+public class ReverseScanIT extends BaseHBaseManagedTimeTableReuseIT {
+    protected static final String ATABLE_INDEX_NAME = generateRandomString();
 
     @BeforeClass
-    @Shadower(classBeingShadowed = BaseHBaseManagedTimeIT.class)
+    @Shadower(classBeingShadowed = BaseHBaseManagedTimeTableReuseIT.class)
```

Comment:
I think you can remove the doSetup() method altogether here. We don't need to disable and delete ATABLE_NAME any more.